subgraph-js/src/buildSubgraphSchema.ts

```diff
@@ -86,7 +66,6 @@ export function buildSubgraphSchema(
     modules,
     new GraphQLSchema({
       query: undefined,
-      directives: [...specifiedDirectives, ...missingFederationDirectives(modules)],
     }),
   );
 
```

Comment:
I believe the comment below this line (where I can't comment...) needs to be changed since now the directives definitions are included in `_service.sdl`?

Reply:
done